tasks/main-clevis.yml

```diff
@@ -4,6 +4,11 @@
     name: "{{ __nbde_client_packages }}"
     state: present
 
+- name: Generate nbde_client dracut config
+  template:
+    src: nbde_client.conf
+    dest: /etc/dracut.conf.d/nbde_client.conf
```

Comment:
what ownership/permissions should this file have?
```
owner: root
mode: u=rw
```
?

Reply:
That would work; `u=rw,g=r,o=r` would be okay as well.